src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,21 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bulldozerDriverAge = 20;
```

Comment:
we should not initialize class variables with declaration, we should accept this value in constructor or setter (fix in all places)
```suggestion
    private int bulldozerDriverAge;
```

Reply:
removed class variable initialization through declaration, but did not add constructors. These are not in this program, but the code would be more verbose with extra constructors.